<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>611</width>
     <height>114</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #2d2d2d;
    color: white;
    border-radius: 10px;
    padding: 6px 12px;
}
QPushButton:hover {
    background-color: #3c3c3c;
}
</string>
   </property>
   <property name="title">
    <string>Functions</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="addBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #4caf50
</string>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icon/add48.png</normaloff>icon/add48.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>48</width>
        <height>48</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="editBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #2196f3</string>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icon/edit48.png</normaloff>icon/edit48.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>48</width>
        <height>48</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="delBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #f44336</string>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icon/del48.png</normaloff>icon/del48.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>48</width>
        <height>48</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="styleSheet">
       <string notr="true">background-color : #fff</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="currentText">
       <string>All</string>
      </property>
      <item>
       <property name="text">
        <string>All</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Regular</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>InService</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="tableStudent">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>1171</width>
     <height>651</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboKey1">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>30</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <item>
    <property name="text">
     <string>ID</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Name</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Department</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="txtKey1">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>30</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search1Btn">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>30</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icon/sreach48.png</normaloff>../icon/sreach48.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtKey2">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>90</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search2Btn">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>90</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icon/sreach48.png</normaloff>../icon/sreach48.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QComboBox" name="comboKey2">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>90</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icon/resources.qrc"/>
 </resources>
 <connections/>
</ui>
